### ISSUE TITLE
Layout/IndentHeredoc with wrong output

### ISSUE THREAD

**Issue by daixiang0** [bbatsov/rubocop#4662]
My ruby code is as below:
```
    stdout = <<~END  # i am sure this line has aligned, 4 spaces at the beginning
2017-03-22 09:28:24 sudo python width_and_depth_tests.py
Width test: PASSED
Depth test: FAILED
END
```
Rubpcop output is:
```
C: Layout/IndentHeredoc: Use 2 spaces for indentation in a heredoc by using <<~ instead of <<~.
```
So I add 2 spaces for 3 lines like below:
```
    stdout = <<~END  # i am sure this line has aligned
  2017-03-22 09:28:24 sudo python width_and_depth_tests.py
  Width test: PASSED
  Depth test: FAILED
END
```
Still same output, then i read the doc and it align with `<<~`, then i try and also fail.
I think the output is not clear or even wrong.

**Comment by pocke** [comment]
This cop says this code should be aligned like the following.


```ruby
    stdout = <<~END
      2017-03-22 09:28:24 sudo python width_and_depth_tests.py
      Width test: PASSED
      Depth test: FAILED
END
```

The 2 space means 2 space * indentation level.
For example:

```ruby
    stdout = <<~END # This line is indented by 2 * 2 space
      2017-03-22 09:28:24 sudo python width_and_depth_tests.py # This line is indented by 2 * 3 space (+1 level)
      Width test: PASSED
      Depth test: FAILED
END
```

**Comment by daixiang0** [comment]
Thanks, i misunderstand it.

**Comment by daixiang0** [comment]
But If i do need a blank in the beginning of EOF iike:
```
<<~EOF

  Hi!
EOF
```
How to fix this cop？

**Comment by pocke** [comment]
Hmm, It is a bug of this cop.
Currently this cop adds an offense for the code, but I think this cop should not add offense.

**Comment by mikegee** [comment]
Let's reconsider the message in this case too: `… by using <<~ instead of <<~.`

**Comment by daixiang0** [comment]
I think so, this message may make people confused.
